security-force-monitor/sfm-cms
# Fixes duplicate tenure issue

## Overview
Fixes an issue we found where an organization's tenure is duplicated when the organization appears in multiple countries.

- fixed duplicate tenure in importer
- added test
- added duplicate organizations to test fixture

### Demo
See single listing ([compared to the staging version, with two](https://back.securityforcemonitor.org/en/organization/view/bda61177-2934-42bf-9293-702d6f073bd3/))
![Screen Shot 2022-03-14 at 11 26 17 AM](https://user-images.githubusercontent.com/38969506/158216849-b6c65f3f-9355-4c5c-bd9b-b5f99defaa5b.png)


## Testing Instructions
- pull down `docket-update` branch
- run new test: `docker-compose -f docker-compose.yml -f tests/docker-compose.yml run --rm app pytest
 tests/test_importer.py::test_no_duplicate_tenures -sxvvv --reuse-db`
- build the app and run migrations
- import ng and lr:
  - `docker-compose --env-file .env.import run --rm app make -e ng_cc_import`
  - `docker-compose --env-file .env.import run --rm app make -e lr_cc_import`
- visit problematic organization and make sure there is no duplicate:
  - http://localhost:8000/en/organization/view/bda61177-2934-42bf-9293-702d6f073bd3/
